packages/backup/src/Helper_Script_Manager.php

```diff
@@ -0,0 +1,238 @@
+<?php
+/**
+ * The Jetpack Backup Helper Script Manager class.
+ *
+ * @package automattic/jetpack-backup
+ */
+
+namespace Automattic\Jetpack\Backup;
+
+/**
+ * Helper_Script_Manager manages installation, deletion and cleanup of Helper Scripts
+ * to assist with backing up Jetpack Sites.
+ */
+class Helper_Script_Manager {
+
+	const RELATIVE_INSTALL_LOCATIONS = array( '', 'wp-content', 'wp-content/uploads' );
```

Comment:
Should we make this dynamic to catch folks using custom wp-content locations with `WP_CONTENT_DIR`?

Reply:
Thank you; that is a great suggestion, and a huge improvement. It's also caused me to update the Helper Script template to make it accept a configurable WordPress install location, instead of assuming it's installed in a subdirectory of it. :)